qemu/tests/cfg/ple_test.cfg

```diff
@@ -0,0 +1,22 @@
+- ple_test:
+    virt_test_type = qemu
+    type = ple_test
+    not_preprocess = yes
+    start_vm = no
```

Comment:
Could you explain the reason to use both `not_preprocess = yes` and `start_vm = no` ?
`not_preprocess = yes` is a bigger level setting.